projects/table-test/src/app/app.module.ts

```diff
@@ -3,12 +3,17 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { ReactiveFormsModule } from '@angular/forms';
 import { AppComponent } from './app.component';
-import { NgxTableModule } from 'table';
+import { NgxTableConfigToken, NgxTableModule } from 'table';
 
 @NgModule({
 	declarations: [AppComponent],
 	imports: [BrowserModule, NgxTableModule, ReactiveFormsModule],
-	providers: [],
+	providers: [
```

Comment:
is `forChild` en `forRoot` nog een ding of is dat geen convention meer?

Reply:
Dat is nog een ding  dat je kan gebruiken inderdaad, maar hier niet echt van toepassing 😅

Reply:
dunno, dan moet je niet manueel die config token providen

maar ik ben al wat te lang weg geweest van angular 😅

Reply:
Oh ja, ik snap waar je naartoe wil, alleen is de volgende stap de component standalone maken waardoor we dus geen module meer hebben en we dus verplicht zijn om het manueel te providen. Dus als we dat nu hadden gedaan hier, moeten we later weer meer refactoren. 😅

Reply:
aha makes sense 👍